pulsar-client-cpp/lib/checksum/crc32c_sse42.cc

```diff
@@ -228,3 +239,12 @@ uint32_t crc32c(uint32_t init, const void *buf, size_t len, const chunk_config *
     DEBUG_PRINTF1("crc = 0x%08x\n", crc);
     return crc;
 }
+
+#else // ! BOOST_ARCH_X86_64
```

Comment:
Does this mean that arm clients can't use crc32c at all?

Reply:
This is only for the SSE 4.2 based implementation, otherwise we fallback to the pure C implementation.